manifests/charts/istio-control/istio-discovery/templates/mutatingwebhook.yaml

```diff
@@ -63,8 +63,12 @@ webhooks:
 {{- end }}
 {{- if .Values.sidecarInjectorWebhook.objectSelector.enabled }}
     objectSelector:
-{{- if .Values.sidecarInjectorWebhook.objectSelector.autoInject }}
       matchExpressions:
+      - key: app
+        operator: NotIn
+        values:
+        - "istiod"
+{{- if .Values.sidecarInjectorWebhook.objectSelector.autoInject }}
       - key: "sidecar.istio.io/inject"
```

Comment:
Maybe we should just put this label on the pod?

Reply:
ah yes, that actually sounds much better

Reply:
Oh, I see that the istiod already has it. the `"sidecar.istio.io/inject"` annotation